index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>[SE] HALFON</title>
+    <title>HALFON by Siege Engineers</title>
```

Comment:
Sorry, this sounds like "Germany's Next Topmodel by Heidi Klum" / "Max Factor by Ellen Betrix" to me :see_no_evil:

Reply:
LOL. Is `HALFON | Siege Engineers` better?

Reply:
Yes, thanks :sweat_smile: